src/action/AsyncActions.ts

```diff
@@ -153,16 +179,16 @@ export function loadVocabulary(
   };
 }
 
-function loadImportedVocabulariesIntoState(
-  vocabularyIri: IRI
-) {
+function loadImportedVocabulariesIntoState(vocabularyIri: IRI) {
   const action = {
     type: ActionType.LOAD_VOCABULARY_IMPORTS,
   };
   return (dispatch: ThunkDispatch, getState: GetStoreState) => {
     dispatch(asyncActionRequest(action, true));
     return Ajax.get(
-      `${getApiPrefix(getState())}/vocabularies/${vocabularyIri.fragment}/imports`,
+      `${getApiPrefix(getState())}/vocabularies/${
```

Comment:
Not a big deal, but the reformatting looks a bit strange - here it is split to multiple rows, some lines up and down the methods params are put to a single row ...

Reply:
If you do not like the formatting, blame Prettier. I had nothing to do with it whatsoever.